DESCRIPTION

```diff
@@ -9,6 +9,7 @@ Authors@R: c(
       person("Hadley", "Wickham", role = c("ctb")),
       person("jQuery contributors", role = c("ctb", "cph"), comment = "jQuery library"),
       person("Leaflet contributors", role = c("ctb", "cph"), comment = "Leaflet library"),
+      person("Leaflet Providers contributors", role = c("ctb", "cph"), comment = "Leaflet Providers plugin"),
```

Comment:
Oops, I was looking at Leaflet, not Leaflet providers. But the point still stands, for the previous line.

Also, I think that Vladimir Agafonkin should be `cph` and `ctb`, and CloudMade should be as `cph`.